### PR TITLE
Connect social

### PR DESCRIPTION
**Pull request by intergalacticspacehighway**
# Why
- To enable social login for users. This PR allows connecting social account (in settings/accounts tab) or login in with social accounts in login screen.
- This PR comments Twitter login logic due to some [limitations](https://showtime-rq88331.slack.com/archives/C02PTNZMEPQ/p1668178647273609?thread_ts=1668103202.406229&cid=C02PTNZMEPQ) in magic regarding accessing the token. 

# How
- Use magic for login and connect account.
- PR is not the cleanest due to redirection involved on web. There is some platform-specific code that can be better abstracted. Will spend some time in the next PRs improving the quality. 

# Test Plan
- Tested on Android, iOS and web.

### CODE REVIEW

File: packages/app/components/login/login-header.tsx [hunk 1]
```diff
@@ -5,8 +5,7 @@ export function LoginHeader() {
   return (
     <View tw="px-4">
       <Text tw="text-center text-sm font-semibold text-gray-900 dark:text-gray-400">
-        Sign in to create and collect free drops from your favorite creators. If
-        this is your first time, you will create a new profile.
+        Sign in to create and collect free drops from your favorite creators.
```
Comment (intergalacticspacehighway) [inline packages/app/components/login/login-header.tsx:8]: shortened this, or else `I already have wallet` gets hidden in some screens

**Review by alantoa — APPROVED** [review]
LGTM 🔥 🔥 🔥 🔥

**Comment by alantoa** [comment]
just tested google and apple social login working for me!

**Review by axeldelafosse — APPROVED** [review]
LFG ✨

**Pull request merged**